var/spack/repos/builtin/packages/py-keyring/package.py

```diff
@@ -20,4 +20,7 @@ class PyKeyring(PythonPackage):
     depends_on('py-setuptools', type='build')
     depends_on('py-setuptools-scm@3.4.1:+toml', type='build')
     depends_on('py-importlib-metadata@3.6:', type=('build', 'run'))
-    # TODO: additional dependencies required for Windows/Linux
+    depends_on('py-secretstorage@3.2:', type='run', when='platform=linux')
+    depends_on('py-jeepney@0.4.1:', type='run', when='platform=linux')
+
```

Comment:
Can you also add a build dep on `py-setuptools-scm@3.4.1:+toml`?

Reply:
Sorry, I am not really sure what you mean, there is already a build dependency on `py-setuptools-scm@3.4.1:+toml` in line 21.

Reply:
Oops, didn't see that. Thanks!